spec/Translator_spec.js

```diff
@@ -12,78 +12,76 @@ describe('Translator', function() {
     var actual = translator.translateToObject(diagnosticsData);
 
     var expected = {
-      diagnostics: {
```

Comment:
I actually had a bug in here. It was getting all the same data for each component.